PWGHF/TableProducer/treeCreatorBplusToD0Pi.cxx

```diff
@@ -93,49 +89,57 @@ DECLARE_SOA_COLUMN(NSigmaTPCTrk1Pi, nSigmaTPCTrk1Pi, float);
 // Events
 DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int);
 DECLARE_SOA_COLUMN(RunNumber, runNumber, int);
-DECLARE_SOA_INDEX_COLUMN_FULL(Candidate, candidate, int, HfCandBplus, "_0");
-DECLARE_SOA_INDEX_COLUMN(McParticle, mcParticle);
-DECLARE_SOA_INDEX_COLUMN(McCollision, mcCollision);
 } // namespace full
 
 // put the arguments into the table
 DECLARE_SOA_TABLE(HfCandBpFulls, "AOD", "HFCANDBPFULL",
+                  collision::BCId,
```

Comment:
In general we could start removing the "full" versions, since the unbound indices are not allowed on hyperloop anymore (but it can be done in a second PR)